BlueShift-iOS-SDK/NetworkReachability.m

```diff
@@ -467,8 +467,8 @@ -(void)reachabilityChanged:(SCNetworkReachabilityFlags)flags
 
 - (NSString *) description
 {
-    NSString *description = [NSString stringWithFormat:@"<%@: %#x (%@)>",
-                             NSStringFromClass([self class]), (unsigned int) self, [self currentReachabilityFlags]];
+    NSString *description = [NSString stringWithFormat:@"<%@: %#lx (%@)>",
+                             NSStringFromClass([self class]), (long int) self, [self currentReachabilityFlags]];
```

Comment:
what is this change?

Reply:
This was optimization for the Xcode warning. The recent version of Xcode started giving warning about "conversion to smaller int" so changed it to long.